AsyncDisplayKit/ASVideoNode.h

```diff
@@ -0,0 +1,30 @@
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+typedef NS_ENUM(NSUInteger, ASVideoGravity) {
+  ASVideoGravityResizeAspect,
+  ASVideoGravityResizeAspectFill,
+  ASVideoGravityResize
+};
+
+@interface ASVideoNode : ASDisplayNode<_ASDisplayLayerDelegate>
+@property (atomic, strong, readwrite) AVAsset *asset;
+@property (nonatomic, assign, readwrite) BOOL shouldAutoPlay;
+@property (atomic) ASVideoGravity gravity;
+@property (atomic) BOOL autorepeat;
+@property (atomic) ASButtonNode *playButton;
+
+- (void)play;
+- (void)pause;
+
+@end
+
+@protocol ASVideoNodeDelegate <NSObject>
+@end
+
+@protocol ASVideoNodeDatasource <NSObject>
+@optional
+- (ASDisplayNode *)playButtonForVideoNode:(ASVideoNode *) videoNode;
```

Comment:
No space after ).  Why does this return an ASDisplayNode rather than ASButtonNode?  Can we expect people to call into the playButton directly and set values on it instead of providing one from the dataSource, or is there a separate use case?